scripts/lib/libvirt/fixtures/cloud-node1-uefi.xml

```diff
@@ -5,8 +5,8 @@
   <vcpu>1</vcpu>
   <os>
     <type arch='x86_64' machine='pc-0.14'>hvm</type>
-<loader readonly='yes' type='pflash'>/usr/share/qemu/ovmf-x86_64-ms-code.bin</loader>
-    <nvram>/var/lib/libvirt/cloud-node1_VARS.fd</nvram>
+<loader readonly='yes' type='pflash'>/usr/share/qemu/ovmf-x86_64-code.bin</loader>
```

Comment:
yep, that sounds like the proper way forward...  I wonder if https://github.com/crowbar/crowbar-core/pull/620 would be any help here (or a fixed version of that one). I actually am digging  trough the uefi deployment path for the aarch64 right now.